src/PlayBack/playBackStyle.ts

```diff
@@ -0,0 +1,59 @@
+import styled from '@emotion/styled';
+
+
+export const PlaybackStyle = styled.div<any>(() => ({
```

Comment:
Doesn't need to be a function since we aren't using `theme` or any other props/logic. Also, the interface can be removed:

```jsx
export const PlaybackStyle = styled.div(({
    boxSizing: 'border-box',
    textAlign: 'center'
}));
```